plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/invalid-package-json/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "yarn-project-with-dep-with-invalid-package-json",
+  "version": "1.0.0",
+  "description": "Yarn test project with yarn.lock.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oss-review-toolkit/ort.git"
+  },
+  "license": "Apache-2.0",
+  "author": "The Author <the.author@mail.example> (https://www.the.author.example/index.html)",
+  "private": true,
+  "dependencies": {
+    "resolve": "1.22.2"
```

Comment:
Can we add a comment next to resolve to indicate that this will
install a malformed package.json, and maybe also add the link to it?

I believe this can save someone the time to try to figure out why this file here is malformed.

Reply:
Are comments in package.json files allowed?
Because comments in JSON are typically considered as invalid.

Reply:
Oh, right ^^